### ISSUE TITLE
enhancement: global search-function for artifacts

### ISSUE THREAD

**Issue by afischer211** [ctron/package-drone#114]
I want to search for artifacts in different channels based on artifactId or type or filename.
A global search function with a simple Web-GUI would be a good solution.

**Comment by ctron** [comment]
There is already a wiki page about a possible query language: https://github.com/ctron/package-drone/wiki/Proposal-for-Query-Language

**Comment by afischer211** [comment]
That is right (I have already extended this wiki page with some words). But we need also a Web-GUI (or a REST-API) for sending a query-request.
